File: lang/fib.py
from logging import basicConfig, info, INFO


def main():
    fib = None
    def f(n):
        if n <= 2:
            return 1
        else:
            return fib(n - 1) + fib(n - 2)
    fib = f
    info("start")
    info(fib(36))


if __name__ == "__main__":
    basicConfig(format="{relativeCreated:>14.6f}ms {levelname} {module} {message}", style="{", level=INFO)
    main()

File: lang/fib.rs
fn fib(n: i32) -> i32 {
    if n <= 2 {
        1
    } else {
        fib(n - 1) + fib(n - 2)
    }
}

fn main() {
    tracing_subscriber::fmt::init();
    tracing::info!("start");
    let result = fib(36);
    tracing::info!("{result}")
}
